source/s3_client.c

```diff
@@ -43,8 +44,9 @@ static const uint32_t s_s3_max_request_count_per_connection = 100;
 static const uint32_t s_connection_timeout_ms = 3000;
 
 /* TODO Provide analysis on origins of this value. */
-static const double s_throughput_per_vip_gbps = 6.25;
+static const double s_throughput_per_vip_gbps = 3.0;
 static const uint32_t s_num_connections_per_vip = 10;
+static const uint32_t s_max_request_jitter_range = 20;
```

Comment:
Maybe just comment that this is 0.2 * 100, where 100 is the max per VIP, so the next poor bastard doesn't lose their mind in here